src/admin/components/SearchForm.js

```diff
@@ -3,57 +3,68 @@ import { connect } from 'react-redux';
 
 import { fetchAddon } from 'core/api';
 import { loadEntities } from 'core/actions';
+import { SearchFormBase } from 'core/components/Search/SearchForm';
 import { gettext as _ } from 'core/utils';
 
-import 'admin/css/SearchForm.scss';
-import 'admin/css/lib/buttons.scss';
+import 'core/css/SearchForm.scss';
+import 'core/css/lib/buttons.scss';
 
-export class SearchFormBase extends React.Component {
+
+export function mapStateToProps({ api }) {
+  return { api };
+}
+
+export function mapDispatchToProps(dispatch) {
+  return {
+    loadAddon({ api, query }) {
+      return fetchAddon({ slug: query, api })
+        .then(({ entities, result }) => {
+          dispatch(loadEntities(entities));
+          return result;
+        });
+    },
+  };
+}
+
+export class AdminSearchFormBase extends SearchFormBase {
```

Comment:
Shoot, you're right. Sorry this PR has become a bit of a jumble in my brain and I forget some things. I totally meant to kill all inheritance. Thanks!